.github/workflows/check-versions.yaml

```diff
@@ -32,6 +32,30 @@ jobs:
           ./.github/scripts/update-versions.sh
           git diff
 
+      - name: Setup go
+        uses: actions/setup-go@v4.0.0
+
+      - name: Setup crane
+        uses: imjasonh/setup-crane@v0.1
+
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.9'
+
+      - name: Update image tags
+        run: |
+          go install github.com/mikefarah/yq/v4@latest
+
+          sudo apt-get install wget apt-transport-https gnupg lsb-release
+          wget -qO - https://aquasecurity.github.io/trivy-repo/deb/public.key | sudo apt-key add -
+          echo deb https://aquasecurity.github.io/trivy-repo/deb $(lsb_release -sc) main | sudo tee -a /etc/apt/sources.list.d/trivy.list
+          sudo apt-get update
+          sudo apt-get install trivy
+          sudo pip install ruamel.yaml dict_deep
```

Comment:
Shouldn't we use a requirements.txt so we can configure dependabot to keep these deps updated? This is likely to break if those deps change.

https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#package-ecosystem

Reply:
We could. Where should we put the file so that dependabot could find them though?

Reply:
You put it in the same folder as the python script.

See:

https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file

I think it should be using pip as package manager and you can define the path to the folder containing the python code and requirements txt.